### PR TITLE
added system settings section

### PR DESCRIPTION
**Pull request by dino-saurabh**
Added the system settings section (aka OpenMRS setting and Global properties) section to the docs as we had a discussion on the OpenMRS talk !!

### CODE REVIEW

**Comment by dino-saurabh** [comment]
@Akayeshmantha please review this is the global properties section that @bmamlin suggested to add to the docs .

File: source/includes/SystemSettings/system_settings.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+```
+* Update a System Setting with given UUID, this method only modifies properties in the request. If the user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *property* | `String` | the property name for this System Setting, names can be up to 255 chars, must be unique, and follow a convention of module ID followed by category & name lower camelCase separated by periods for e.g., addresshierarchy.allowFreetext
+    *description* | `String` | a description for the usage of this property
+    *datatypeClassname* | `String` | Data type for this System Setting.OpenMRS provides Custom data type resource, which gives flexibility to select the data type accordingly             *datatypeConfig* | `String` | An optional identifier from the fulfiller (e.g., lab            *preferredHandlerClassname*  | `String` | Handler subresource for the Custom Data Type used. Can optionally define a specific handler class wants to use (otherwise, the framework will choose the best handler for the chosen DataType)
```
Comment (Akayeshmantha) [inline source/includes/SystemSettings/system_settings.md]: I can see some kind of formatting issue here.

File: source/includes/SystemSettings/system_settings.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+# System Setting
+
+## System Setting Overview
+
+* System Settings are used to store module and system-wide settings. They primarily consist of a property name and a value and a description explaining how this property is being used
```
Comment (Akayeshmantha) [inline source/includes/SystemSettings/system_settings.md]: make sure to have .  in the end of sentences.

**Review by Akayeshmantha — CHANGES_REQUESTED** [review]
Overall good work. Few changes.

**Comment by dino-saurabh** [comment]
@Akayeshmantha  these are the example calls I made and the responses I got from the server while testing the rest API endpoints

[systemSettingsCurlchecking.txt](https://github.com/openmrs/openmrs-contrib-rest-api-docs/files/4741763/systemSettingsCurlchecking.txt)

File: source/includes/SystemSettings/system_settings.md (hunk 2 above)
Comment (dino-saurabh) [reply source/includes/SystemSettings/system_settings.md]: @Akayeshmantha Ahh sorry I will refactor it .

File: source/includes/SystemSettings/system_settings.md (hunk 1 above)
Comment (dino-saurabh) [reply source/includes/SystemSettings/system_settings.md]: @Akayeshmantha  I also thought so but the view is fine when I build it should I still refactor it??

**Review by Akayeshmantha — APPROVED** [review]
Good work

**Pull request merged**